docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  nix:
+    image: nixos/nix:2.6.1
+    volumes:
+      - ./:/usr/local/src
+    working_dir: /usr/local/src/
+    entrypoint: nix --extra-experimental-features "nix-command flakes"
```

Comment:
Most folks don't have Nix installed on their machines, and GitHub doesn't have NixOS runners, so using a containerized build environment makes sense.

It's worth calling attention to the entry point. Nix flakes are still considered an "experimental" feature. However, they will eventually replace "channels," which is the original way Nix handled packaging.

Reply:
👍